packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -92,7 +112,7 @@ export const MobileComboBox = React.forwardRef(function MobileComboBox<T extends
         ref={domRef}
         includeNecessityIndicatorInAccessibilityName>
         <ComboBoxButton
-          {...mergeProps(triggerProps, fieldProps)}
+          {...mergeProps(triggerProps, fieldProps, {onBlur: trayInputProps.onBlur, onFocus: trayInputProps.onFocus, autoFocus: props.autoFocus})}
```

Comment:
Fixes the following bugs:
- Mobile combobox isn't autofocused when autoFocus is applied
- Mobile combobox opens when user clicks the buttons to change the controlled input value (see story)

Root issue is that the mobile combobox button doesn't have the blur/focus handlers that `useComboBox` returns so `state.isFocused` doesn't get updated when blurring/focusing the button

Reply:
Rob found an interesting bug where occasionally a mobile combobox with `menuTrigger=focus` will open and automatically close on tap. Did some digging and I believe this is due to the tray opening on pressdown (focus opens the tray due to `menuTrigger=focus`) and the subsequent press up triggers a onInteractOutside causing the tray to close. Still trying to think about how to handle this, but kinda at a loss (almost wanna say that `menuTrigger=focus` shouldn't work on mobile). 

100% reproduction steps:
1. Go to combobox story w/ `menuTrigger = autofocus` 
2. Make sure the combobox isn't focused by tapping on the background
3. Tap and hold on the combobox. Notice the tray opens
4. Release your press. Note that the tray closes

Reply:
I think it just doesn't make any sense for it to open on press start, makes more sense to open on press end like when we handle touch for menus
It might be weird, but can focus occur on pressend instead?

Reply:
yep, that was going to be my next experiment

Reply:
I can no longer reproduce :)